### DISCUSSION TITLE
Improve SHM Transport logging messages

### DISCUSSION THREAD

**Discussion by JLBuenoLopez** [eProsima/Fast-DDS#3578] [Ideas]
Some opened tickets are complaining that the SHM Transport initialization failure logged messages are unclear from a user point of view. For instance #2790, #2003.

I am opening this ticket to keep track about improving the messages so they can be more helpful to the user.

**Comment by aditya2592** [comment]
I had a question about the `fastdds shm clean` command and the code in `clean.py`. On my ubuntu system, FastDDS with SHM generates the folllowing file types
```
-rw-r--r--  1 root root        32 Jan 19 21:16 sem.fastrtps_port7438_mutex
-rw-r--r--  1 root root         0 Jan 19 21:16 fastrtps_port7439_el
-rw-r--r--  1 root root     52416 Jan 19 21:16 fastrtps_port7439
```
However `clean.py` only cleans up files ending in `_el` and files like `fastrtps_port7439` are left behind. Is there a reason behind that?
